test/language/expressions/this/S11.1.1_A3.1.js

```diff
@@ -17,7 +17,7 @@ if (MyFunction() !== this) {
 }
 
 //CHECK#2
-function MyFunction() {return eval("this")}
-if (MyFunction() !== this) {
-  $ERROR('#2: function MyFunction() {return eval("this")} MyFunction() === this. Actual: ' + (MyFunction()));
+function MyFunction2() {return eval("this")}
```

Comment:
is `MyFunction` some sort of standard in the repo?  Is there any reason not to make these a little more readable (hah in test262?!) with names like `ReturnsThis` and `ReturnsEvalThis` or something?